Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,41 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
```

Comment:
Indeed (another thing that could be added to a PR for `outdated`).

These kinds of issues should in theory be flagged by `brew style <filename>`, which gives good advise most of the time and should be taken into account for new code. Unfortunately, running it on all of Homebrew is currently not very helpful, because we've never did a mass cleanup of our code after `brew style` was introduced.